src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -526,29 +532,39 @@ private static CodeAction TryCreateCodeAction(Document document, AddImportFixDat
                 _ => throw ExceptionUtilities.Unreachable,
             };
 
-        private static ITypeSymbol GetAwaitInfo(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
+        private static ITypeSymbol? GetAwaitInfo(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
         {
             var awaitExpression = FirstAwaitExpressionAncestor(syntaxFactsService, node);
 
             var innerExpression = syntaxFactsService.GetExpressionOfAwaitExpression(awaitExpression);
 
+            if (innerExpression is null)
+            {
+                return null;
+            }
+
             return semanticModel.GetTypeInfo(innerExpression).Type;
         }
 
-        private static ITypeSymbol GetCollectionExpressionType(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
+        private static ITypeSymbol? GetCollectionExpressionType(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
         {
             var collectionExpression = FirstForeachCollectionExpressionAncestor(syntaxFactsService, node);
 
+            if (collectionExpression is null)
```

Comment:
This is the real fix. Callers to `GetCollectionExpressionType` already handle when it returns null.